packages/remote-config/lib/index.d.ts

```diff
@@ -278,20 +297,80 @@ export namespace FirebaseRemoteConfigTypes {
    */
   export class Module extends FirebaseModule {
     /**
-     * The number of milliseconds since the last Remote RemoteConfig fetch was performed.
+     * Indicates the value in milliseconds set for the minimum interval that needs to elapse
+     * before a fetch request can again be made to the Remote Config backend.
      */
-    lastFetchTime: number;
+    minimumFetchIntervalMillis: number;
```

Comment:
`minimumFetchIntervalMillis ` isn't on the root on the Web SDK, it's only accessible via settings